### ISSUE TITLE
logical-backup image not arm64 compatible

### ISSUE THREAD

**Issue by tomsiewert** [zalando/postgres-operator#2313]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** `ghcr.io/zalando/postgres-operator:v1.10.0`
- **Which image of logical-backup are you using?** `registry.opensource.zalan.do/acid/logical-backup:v1.10.0`
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?** K3s
- **Are you running Postgres Operator in production?** yes
- **Type of issue?** Bug / Feature request

The logical-backup image is not working on arm64 as it's being built with an amd64 base. See:
```
➜ k logs -n matrix jobs/logical-backup-matrix-postgres-28054110 
Found 7 pods, using pod/logical-backup-matrix-postgres-28054110-7jt7x
exec /dump.sh: exec format error
```

**Comment by FxKu** [comment]
Yeah - so far only Spilo and operator have arm support. We plan to follow up with the rest of the images - UI, pooler and logical backup in the upcoming weeks.

**Comment by mprimeaux** [comment]
Is there any help I can provide for this enhancement?

**Comment by agaffney** [comment]
Has there been any progress on this?

**Comment by Alwin-Stockinger** [comment]
Any update on this?

**Comment by lyz-code** [comment]
@FxKu can we track the fact that the pooler is not arm compatible either in this issue? or should I open a new one?

**Comment by acelinkio** [comment]
ARM64 images for postgres-operator & postgres-operator-ui are not present on registry.opensource.zalan.do.  But do exist on ghcr.io.

Should the default in the helm charts points point to the ghcr.io then?

**Comment by davidfrickert** [comment]
would be nice to see `arm64` images of `registry.opensource.zalan.do/acid/pgbouncer:master-32`

**Comment by FxKu** [comment]
Logical backup has arm support for a long time and pooler was just added with v2.0 release.
